tests/PublicSuffixListTest.php

```diff
@@ -28,36 +28,15 @@ class PublicSuffixListTest extends TestCase
      */
     private $list;
 
-    private $dataDir;
-
-    protected function setUp()
-    {
-        parent::setUp();
-        $this->list = new PublicSuffixList();
-        $this->dataDir = dirname(__DIR__) . '/data';
-    }
-
-    public function testConstructorThrowsTypeError()
-    {
-        $this->expectException(TypeError::class);
-        new PublicSuffixList(new \stdClass());
-    }
-
-    public function testConstructorThrowsInvalidArgumentException()
-    {
-        $this->expectException(InvalidArgumentException::class);
-        new PublicSuffixList('/usr/bin/foo/bar');
-    }
-
-    public function testConstructorWithFilePath()
+    public function setUp()
     {
-        $this->assertEquals($this->list, new PublicSuffixList($this->dataDir . '/' . PublicSuffixListManager::PDP_PSL_PHP_FILE));
+        $manager = new PublicSuffixListManager(new FileCacheAdapter(), new CurlHttpAdapter());
+        $this->list = $manager->getList();
     }
 
-    public function testConstructorWithArray()
+    public function testGetRules()
     {
-        $rules = include $this->dataDir . '/' . PublicSuffixListManager::PDP_PSL_PHP_FILE;
-        $this->assertEquals($this->list, new PublicSuffixList($rules));
+        $this->assertInternalType('array', $this->list->getRules());
```

Comment:
I don't really know if this test is needed of even if the `getRules` method on its own is now needed since it returns a raw array which is not useful outside of a PublicSuffixList object